generic/tests/vlan.py

```diff
@@ -188,6 +203,22 @@ def flood_ping(src, dst):
             set_ip_vlan(session, vlan_i, v_ip, ifname[vm_index])
         set_arp_ignore(session)
 
+    if params["os_type"] == "windows":
+        for vm_index, vm in enumerate(vms):
+            status, output = utils_test.ping(dest=vm_ip[(vm_index + 1) % 2], count=10,
+                                             session=session_ctl[vm_index],
+                                             timeout=30)
+            loss = utils_test.get_loss_ratio(output)
+            if not loss and ("TTL=" in output):
```

Comment:
@sitoliu Could you please explain why "TTL=" is needed to show in the output? Thanks